policyengine_us/parameters/gov/states/ri/tax/income/adjusted_gross_income/subtractions/subtractions.yaml

```diff
@@ -0,0 +1,19 @@
+description: Rhode Island subtractions these items from federal AGI when computing Rhode Island AGI.
+
+metadata:
+  reference:
+  - title: 2021 RI Schedule M - page 1 RI Modifications to Federal AGI 
+    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-01/2021-ri-schedule-m_w.pdf#page=1
+  - title: 2022 RI Schedule M - page 1 RI Modifications to Federal AGI 
+    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%20RI%20Schedule%20M_w.pdf#page=1
+  - title: § 44-30-12.  Rhode Island income of a resident individual. 
+    href: http://webserver.rilin.state.ri.us/Statutes/title44/44-30/44-30-12.HTM
+  period: YEAR
+  label: List of Rhode Island AGI subtractions
+
```

Comment:
need unit: list - look in the doc for the format of a list parameter

Reply:
ok! I think the correct link in the doc is https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/gov/states/dc/dhs/tanf/income/sources/earned.yaml